ecs.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $containerConfigurator->import('vendor/sylius-labs/coding-standard/ecs.php');
+};
```

Comment:
Missing empty line at the end of the file

https://stackoverflow.com/questions/36019534/add-an-empty-line-at-end-of-file-according-to-psr-2-on-phpstorm